google-cloud-clients/google-cloud-spanner/pom.xml

```diff
@@ -150,6 +150,12 @@
       <artifactId>guava-testlib</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Hi @elharo, is it okay to import this lib in `google-cloud-java` spanner? I would like to test the setting on an environment variable. Can I ask what the best practice would be for this case? Thanks.

Reply:
It's test scope, so this is not a big deal. I would still consider whether you truly need this. If it's only a minor convenience I wouldn't add it.